(DOCSP-45646) Pillar content (#47)
mongodb/docs-atlas-architecture: source/security.txt

.. default-domain:: mongodb

The Security pillar of the {+service+} {+waf+} includes features that
ensure data integrity and privacy through comprehensive safeguarding
measures, such as encryption at rest and in transit, fine-grained access
controls, and adherence to regulatory standards.

Use the following {+atlas-arch-center+} resources to learn about
security in {+service+}:

.. card-group::
   :columns: 2
   :style: extra-compact

   .. card::
      :headline: Network Security
      :url: https://mongodb.com/docs/manual/data-modeling/
      :icon: general_security
      :icon-alt: Atlas security icon

>       |service-fullname| enforces mandatory |tls| encryption for database connections.
>       Restrict database access to specific IP addresses and |cidr| ranges
>       and use private endpoints for staging and production projects.

Reviewer: Let's remove the first line here to keep the cards ~4 lines each for consistency of appearance
```suggestion
      Restrict database access to specific IP addresses and |cidr| ranges
      and use private endpoints and VPC peering.
```

   .. card::
      :headline: Authorization and Authentication
      :url: https://mongodb.com/docs/TBD/setup-cluster-security/
      :icon: general_action_login
      :icon-alt: Atlas login icon

>       Use Role-Based Access Control (RBAC) with built-in and custom roles
>       to control operations on resources. Implement RBAC with the principle of
>       least privilege and integrate with identity providers in |service|.

Reviewer: ```suggestion
      Use Role-Based Access Control (RBAC) with built-in and custom roles
      to control operations on resources. Get recommendations on the best authentication method for production environments.
```

   .. card::
      :headline: Data Encryption
      :url: https://mongodb.com/docs/TBD/auth/
      :icon: general_security_encryption
      :icon-alt: Atlas encryption icon

      Use encryption features in |service| to protect data while in transit,

>       at rest, and in use, and to safeguard data through its full lifecycle.

Reviewer: ```suggestion
      at rest, and in use to safeguard data through its full lifecycle.
```

   .. card::
      :headline: Compliance
      :url: https://mongodb.com/docs/TBD/compliance/
      :icon: general_security_hipaa_compliance
      :icon-alt: Atlas HIPAA compliance icon

      |service| meets a list of compliance standards to ensure that you

>       can protect and monitor data.

Reviewer: ```suggestion
      can protect and monitor data while complying with external requirements.
```

   .. card::
      :headline: Auditing and Logging
      :url: https://mongodb.com/docs/TBD/auditing-logging/
      :icon: general_features_realtime
      :icon-alt: Atlas realtime reporting icon

>       Use auditing and logs to monitor and log |service| platform activities.

Reviewer: ```suggestion
      Use auditing and logs to monitor and log |service| platform activities. Get recommendations on the events you should audit.
```
